content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -111,18 +118,29 @@ webhooks:
     - CREATE
     resources:
     - pods
+    scope: "Namespaced"
   clientConfig:
     service:
       namespace: <namespace of the front-end service>
       name: <name of the front-end service>
+      port: 443
     caBundle: <pem encoded ca cert that signs the server cert used by the webhook>
+  admissionReviewVersions:
+  - v1beta1
+  timeoutSeconds: 1
 ```
 
 {{< note >}}
 When using `clientConfig.service`, the server cert must be valid for
 `<svc_name>.<svc_namespace>.svc`.
 {{< /note >}}
 
+{{< note >}}
+Default timeout if not specified is 30 seconds but it is encouraged to use a very
```

Comment:
is it worth mentioning the field exists since 1.14?

Reply:
rephrased.